projects/step-frontend/src/lib/modules/grid/components/status-summary/status-summary.component.html

```diff
@@ -0,0 +1,37 @@
+<div
```

Comment:
This component looks very similar to `status-distribution.component`
Probably we can unify them or create a set of similar components in the same approach.